### ISSUE TITLE
Be able to edit data irrespective of who has entered it

### ISSUE THREAD

**Issue by jonparsonsjncc** [jonparsonsjncc/nbmp#24]
As a: Internal admin user
I want to: be able to edit data irrespective of who has entered it
So that I can: maintain the accuracy of the data

**Comment by jonparsonsjncc** [comment]
yes in response to a suggestion from a volunteer of an inaccuracy

**Comment by mnewmanBats** [comment]
Philip suggested improved validation may help

**Comment by jonparsonsjncc** [comment]
could be helped with better data validation at point of data entry - ie tying in the data selected with the recording period

**Comment by mnewmanBats** [comment]
Comment admin not users be allowed to
